ipaserver/install/server/install.py

```diff
@@ -940,9 +940,24 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
     print("")
-    print("Next steps:")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("\nObtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
```

Comment:
Either add ``\n`` to the previous line or add an empty print() statement. I prefer the latter.

Reply:
Cannot understand this. Because something this sort is logged after install.

IPA server installation successful

You can access Web UI using https://rhel7u4-6.gsslab.pnq2.redhat.com/ipa/ui

Obtain a Kerberos Ticket for issuing IPA command 'kinit admin'.


There is already an line displayed between them.

Reply:
My review comment was solely about the placement of ``\n`` in your patch.
Semantically the newline char ``\n`` belongs to the previous line, not the next line. This comes from the era of [typewriters](https://youtu.be/oSyivQLQfQw?t=43s), where a new line followed by a carriage return terminated a line. An example will probably help more:

Instead of:
```
    print("text")
    print("\nmore test")
```

it is better to use
```
    print("text\n")
    print("more test")
```

or even better

```
    print("text")
    print()
    print("more test")
```

All three variants will print out the same text. I prefer the last variant because it makes the code look like the actual output.